<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationInfoWidget</class>
 <widget class="QWidget" name="SimulationInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>211</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Rendering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="targetFPSLabel">
        <property name="text">
         <string>Target FPS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actualFPSLabel">
        <property name="text">
         <string>Actual FPS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="renderTimeLabel">
        <property name="text">
         <string>Render time:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Simulation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="targetScaleLabel">
        <property name="text">
         <string>Target scale:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actualScaleLabel">
        <property name="text">
         <string>Actual scale:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="updateTimeLabel">
        <property name="text">
         <string>Update time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="simulationTimeLabel">
        <property name="text">
         <string>Simulation:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
